@navikt/core/react/src/layout/box/Box.tsx

```diff
@@ -0,0 +1,175 @@
+import React, { forwardRef } from "react";
+import { OverridableComponent } from "../../util/OverridableComponent";
+import cl from "clsx";
+import {
+  ResponsiveProp,
+  SpacingScale,
+  getBorderRadius,
+  getResponsivePropsPaddingForInlineOrBlock,
+} from "../utilities/css";
+import {
+  BorderRadiusSpecifier,
+  BorderColorSpecifier,
+  BackgroundSpecifier,
+  ShadowSpecifier,
+} from "./types";
+
+type Spacing = ResponsiveProp<SpacingScale>;
+
+export interface BoxProps extends React.HTMLAttributes<HTMLDivElement> {
+  children?: React.ReactNode;
+  /** Background color */
+  background?: BackgroundSpecifier;
+  /** Border color */
+  borderColor?: BorderColorSpecifier;
+  /** Border radius */
+  borderRadius?: BorderRadiusSpecifier;
+  /** Spacing around children. Accepts a spacing token or an object of spacing tokens for different screen sizes.
```

Comment:
Burde vi skrive "breakpoints" i stedet for "screen sizes"? Bare en tanke.

```suggestion
  /** Spacing around children. Accepts a spacing token or an object of spacing tokens for different breakpoints.
```